ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -95,7 +95,21 @@ chain:
       openshift-tests run ${TEST_SUITE} ${TEST_ARGS:-} \
         --provider "${TEST_PROVIDER}" \
         -o "${ARTIFACT_DIR}/e2e.log" \
-        --junit-dir "${ARTIFACT_DIR}/junit"
+        --junit-dir "${ARTIFACT_DIR}/junit" 2>&1 | tee /tmp/openshift-tests.log
+      
+      exit_code=$?
```

Comment:
After looking at the logs, I still see failures but the conformance test result is success. I think we're getting the exit code of the `tee` command and not the `openshift-tests` command

look at the example below. The first exit code is 1 and the second is 0 despite the two greps failing in both examples. 

```
#!/bin/bash
#
echo "test" | grep "madeup"
echo $?

echo "test" | grep "madeup" | tee /tmp/somefile
echo $?
```

Reply:
i'm now retrieving the exit code from `${PIPESTATUS[0]}`, which is accurately the exit code of `openshift-tests`, not `tee`.